pycbc/types/timeseries.py

```diff
@@ -838,14 +838,14 @@ def to_frequencyseries(self, delta_f=None):
         return f
 
     def add_into(self, other):
-        """Return the sum of the two time series accounting for the time stamp.
+        """Add another time series into this one where they overlap.
 
         The other vector will be resized and time shifted with sub-sample
```

Comment:
The docstring should make it clear that this method returns a new object and does not modify self.

Also, the name of the method is counterintuitive: you would think that `a.add_into(b)` would mean that `a` is time shifted and added to `b`, but the method does the opposite.

Reply:
The method name `insert` runs the risk of being confused with the `insert` method on builtin Python container types, which behaves very differently. How about `add_from`?

Reply:
Changed to 'inject' as this is a synonym and accurate about what it is doing.